recipe/meta.yaml

```diff
@@ -12,8 +12,11 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  script:
+    - export DISABLE_NUMCODECS_AVX2=""  # [unix]
+    - set DISABLE_NUMCODECS_AVX2=""  # [win]
```

Comment:
Was looking into some other options here and found that `-march=nocona -mtune=generic` might be a good combination. This ensures that the generated library will run on quite ancient architecture if needed, but will support AVX2 or other instructions on newer processors if available.